ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -150,6 +148,9 @@ tests:
     - as: must-gather-e2e-test
       cli: latest
       commands: make test-must-gather-e2e
+      dependencies:
+      - env: OO_BUNDLE
+        name: ci-index
```

Comment:
this is because this lane needs to deploy the operator, so we have an indirect dependency on the bundle, right?

Reply:
Yes. 
I don't know how it worked before, it was set differently.